README.md

```diff
@@ -12,7 +12,7 @@ We are also looking for help from designers and 3D modelers! Please send an emai
 
 ## Development 
 
-To successfully run the project, you will need to recreate one file that is gitignored for security reasons, `/src/constants/apiKeys.js`. Follow the instructions in the template located at `/src/constants/apiKeysTemplate.js`. After that, just run
+Just run
```

Comment:
Is the specification that api keys need to be in the .env file present in the houston docs? If not, it ought to be mentioned herein.

Reply:
I can't believe we don't have docs already about settings :cold_sweat: but here it is now https://github.com/WildMeOrg/houston/pull/702

Reply:
Thank you!